ams/server/src/main/java/com/netease/arctic/server/dashboard/model/TableOptimizingProcess.java

```diff
@@ -125,4 +167,40 @@ public Long getTargetChangeSnapshotId() {
   public void setTargetChangeSnapshotId(Long targetChangeSnapshotId) {
     this.targetChangeSnapshotId = targetChangeSnapshotId;
   }
+
+  public FilesStatistics getInputFiles() {
+    return inputFiles;
+  }
+
+  public void setInputFiles(FilesStatistics inputFiles) {
+    this.inputFiles = inputFiles;
+  }
+
+  public FilesStatistics getOutputFiles() {
+    return outputFiles;
+  }
+
+  public void setOutputFiles(FilesStatistics outputFiles) {
+    this.outputFiles = outputFiles;
+  }
+
+  public void init() {
+    if (finishTime > 0) {
+      duration = finishTime - startTime;
+    } else {
+      duration = System.currentTimeMillis() - startTime;
+    }
+    duration = duration > 0 ? duration : 0;
+    FilesStatisticsBuilder inputBuilder = new FilesStatisticsBuilder();
+    FilesStatisticsBuilder outputBuilder = new FilesStatisticsBuilder();
+    if (summary != null) {
+      inputBuilder.addFiles(summary.getEqualityDeleteSize(), summary.getEqDeleteFileCnt());
+      inputBuilder.addFiles(summary.getPositionalDeleteSize(), summary.getPosDeleteFileCnt());
+      inputBuilder.addFiles(summary.getRewriteDataSize(), summary.getRewriteDataFileCnt());
+      inputBuilder.addFiles(summary.getRewritePosDataSize(), summary.getReRowDeletedDataFileCnt());
+      outputBuilder.addFiles(summary.getNewFileSize(), summary.getNewFileCnt());
+    }
+    inputFiles = inputBuilder.build();
+    outputFiles = outputBuilder.build();
+  }
```

Comment:
Maybe we don't need this "init" method. We can directly parse from the summary when the "getOutputFiles" method is called.

Reply:
If we directly parse from the summary when the `getOutputFiles` method is called, the field `outputFiles` is useless, so we can remove it (the same as `inputFiles`).

So, there are two feasible designs:
 1. keep the `init` method, and keep the field `inputFiles` and `outputFiles`, 
 2. remove the `init` method, and remove the filed `inputFiles` and `outputFiles` but keep the `getXXX` method

I prefer 1, because it seems to be more clear as a class for a HTTP interface. What do you think? @shidayang

Reply:
I have changed it.